models/app.models.js

```diff
@@ -33,10 +33,10 @@ exports.selectArticleById = (article_id) => {
   });
 };
 
-exports.selectArticles = () => {
+exports.selectArticles = (sort_by = "created_at", order = "desc") => {
   let queryString = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) as comment_count FROM articles`;
   queryString += ` LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id`;
-  queryString += ` ORDER BY created_at DESC`;
+  queryString += ` ORDER BY ${sort_by} ${order.toUpperCase()}`;
```

Comment:
psql will reject the sort_by if it's not a column it can find - but because you are directly using the value here you are open to sql injection. if someone did `/api/articles?sort_by=DELETE * FROM articles;` for example, you're open to that possibility here.

you want to greenlist the options for both sort_by and order in this model, create an array of the options you say are okay and check whether the passed in value is included. if not, promise.reject it, otherwise let psql do it's thing